Package/PluginInstaller.cs

```diff
@@ -455,7 +455,16 @@ internal static List<string> UnpackPackage(string packagePath, string destinatio
                             {
                                 throw;
                             }
-                            catch
+                            catch (IOException ex) when (ex.Message.Contains("There is not enough space on the disk"))
+                            {
+                                log.Error(ex.Message);
+                                var req = new AbortOrRetryRequest("Not enough disk space", $"File '{part.FullName}' requires {Utils.BytesToReadable(part.Length)} of free space. " +
```

Comment:
```suggestion
                                var req = new AbortOrRetryRequest("Not Enough Disk Space", $"File '{part.FullName}' requires {Utils.BytesToReadable(part.Length)} of free space. " +
```

Here we can use title case.